concepticondata/conceptlists/Lai-2023-291.tsv

```diff
@@ -0,0 +1,292 @@
+Lai-2023-291-86	233	the wheat	1077	WHEAT	410	the wheat
+Lai-2023-291-87	359	the work	1086	WORK (ACTIVITY)		the work
+Lai-2023-291-88	169	salty	1091	SALTY	1076	salty
+Lai-2023-291-89	127	middle	1093	MIDDLE	708	middle
+Lai-2023-291-90	159	to reside (live)	363	DWELL (LIVE, RESIDE)	1452	to reside (live)
+Lai-2023-291-91	376	to cook	1100	COOK (SOMETHING)		to cook
+Lai-2023-291-92	375	to climb	1102	CLIMB		to climb
+Lai-2023-291-93	279	painful	1129	PAINFUL		painful
+Lai-2023-291-94	218	the thunder	1150	THUNDER	8	the thunder
+Lai-2023-291-95	397	to open	1156	OPEN		to open
+Lai-2023-291-96	178	to shoot (an arrow)	1172	SHOOT	1611	to shoot (an arrow)
+Lai-2023-291-97	247	yesterday	1174	YESTERDAY	739	yesterday
+Lai-2023-291-98	124	many	1198	MANY	987	many
+Lai-2023-291-99	237	the wife	1199	WIFE	248	the wife
+Lai-2023-291-100	324	the husband	1200	HUSBAND		the husband
+Lai-2023-291-101	52	the feather	1201	FEATHER	299	the feather
+Lai-2023-291-102	11	big	1202	BIG	964	big
+Lai-2023-291-103	120	long	1203	LONG	972	long
+Lai-2023-291-104	298	the bark	1204	BARK		the bark
+Lai-2023-291-105	222	the tongue	1205	TONGUE	139	the tongue
+Lai-2023-291-106	249	young	1207	YOUNG	1059	young
+Lai-2023-291-107	443	I [first person singular]	1209	I		I [first person singular]
+Lai-2023-291-108	87	heavy	1210	HEAVY	1014	heavy
+Lai-2023-291-109	446	thou [second person singular]	1215	THOU		thou [second person singular]
+Lai-2023-291-110	165	the rope	1218	ROPE	619	the rope
+Lai-2023-291-111	360	the worm	1219	WORM		the worm
+Lai-2023-291-112	206	the tail	1220	TAIL	267	the tail
+Lai-2023-291-113	146	the nose	1221	NOSE	80	the nose
+Lai-2023-291-114	86	the heart	1223	HEART	144	the heart
+Lai-2023-291-115	119	the liver	1224	LIVER	145	the liver
+Lai-2023-291-116	305	the day (opposed to night)	1225	DAY (NOT NIGHT)		the day (opposed to night)
+Lai-2023-291-117	245	the year	1226	YEAR	777	the year
+Lai-2023-291-118	44	the earth (soil)	1228	EARTH (SOIL)	40	the earth (soil)
+Lai-2023-291-119	36	dirty	1230	DIRTY	1086	dirty
+Lai-2023-291-120	142	new	1231	NEW	1050	new
+Lai-2023-291-121	426	warm (hot)	1232	WARM		warm (hot)
+Lai-2023-291-122	339	the night	1233	NIGHT		the night
+Lai-2023-291-123	191	smooth	1234	SMOOTH	1037	smooth
+Lai-2023-291-124	236	who	1235	WHO	953	who
+Lai-2023-291-125	232	what	1236	WHAT	954	what
+Lai-2023-291-126	234	where	1237	WHERE	955	where
+Lai-2023-291-127	267	how	1239	HOW		how
+Lai-2023-291-128	259	few	1242	FEW		few
+Lai-2023-291-129	210	thick	1244	THICK	980	thick
+Lai-2023-291-130	188	small	1246	SMALL	965	small
+Lai-2023-291-131	42	the ear	1247	EAR	81	the ear
+Lai-2023-291-132	49	the eye	1248	EYE	79	the eye
+Lai-2023-291-133	9	the belly	1251	BELLY	96	the belly
+Lai-2023-291-134	97	the house	1252	HOUSE	494	the house
+Lai-2023-291-135	84	the head	1256	HEAD	74	the head
+Lai-2023-291-136	239	the wing	1257	WING	298	the wing
+Lai-2023-291-137	183	to sing	1261	SING	1184	to sing
+Lai-2023-291-138	90	high / tall	1265	HIGH	968	high / tall
```

Comment:
[HIGH OR TALL](https://concepticon.clld.org/parameters/2790) perhaps is a more suitable mapping.